### PR TITLE
PCM-681

### PR DESCRIPTION
**Pull request by renujhamtani**
@ardasgup @sshumake Please review and merge.

### CODE REVIEW

**Comment by renujhamtani** [comment]
@pbathia Could you please review this.

File: app/cases/views/confirmCaseSeverityChangeModal.jade [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.modal-header
+  h3(translate='') Changing Case Severity
+.modal-body
+  .row
+    .col-sm-12
+      div Are you sure you want to change this case severity to {{CaseService.kase.severity.name}}?
```
Comment (pbathia) [inline app/cases/views/confirmCaseSeverityChangeModal.jade]: @renujhamtani we don't need to translate this sentence 'Are you sure you want to change this case severity to'?

**Comment by pbathia** [comment]
Rest of the code looks good to me.

**Comment by renujhamtani** [comment]
https://projects.engineering.redhat.com/browse/PCM-626

Linked the comments to the translation tags defect.

**Comment by ardasgup** [comment]
@renujhamtani  this code is fine, but after checking the other modals it is my view that these confirmation popups should be a common view with dynamic button, message and header fields to be set from controller. I can see this residing under /common.
But for now as per the story points this is fine. We should create a Jira and address this hardening.

**Comment by renujhamtani** [comment]
Merging! Would need to address the issue mentioned by Aritro in hardening.
@ardasgup Could you please create a JIRA for this for hardening? I am merging this PR.

**Pull request merged**